steps/household/household.tsx

```diff
@@ -138,15 +143,13 @@ const step = {
           key: "member-changes",
           Component: TextAreaDetails,
           props: {
-            summary: "Add note about any changes in household members" as React.ReactNode,
-            label: { value: "Notes" } as {
-              id?: string;
-              value: React.ReactNode;
-            },
-            name: "member-changes-notes"
-          },
-          defaultValue: "",
-          emptyValue: "",
+            summary: "Add note about any changes in household members",
+            label: { value: "Notes" },
+            name: "member-changes-notes",
+            includeCheckbox: true
+          } as TextAreaDetailsProps,
+          defaultValue: { value: "", isPostVisitAction: false },
+          emptyValue: { value: "", isPostVisitAction: false },
```

Comment:
Ah. I've realized that this doesn't work properly. `remultiform` checks for equality, which will never be true for objects, even if they look the same. For now, I think it's ok, but I think we probably need something in the backlog to support this properly in `remultiform`. That means also adding it to `ComponentValue` and reverting the change above to `ProcessStepDefinition`.